incuna-sass/_mixins.sass

```diff
@@ -3,3 +3,8 @@
 @import mixins/placeholder
 @import mixins/rgba-background
 @import mixins/svg-background
+@import mixins/svg
+// If your project does not have sprites folders set up
+// the following may result in an error when you compile
+// your sass
+@import mixins/sprites
```

Comment:
Sorry @hippogriffic, by "leave it in and comment" I thought you meant `// @import mixins/sprites`. We need it commented out to avoid the errors: projects can import it separately (as they currently do) to use it.